corehq/apps/sms/models.py

```diff
@@ -2297,3 +2290,6 @@ def has_migration_completed(cls, name):
             return True
         except cls.DoesNotExist:
             return False
+
+
+from corehq.apps.sms.signals import *
```

Comment:
yeah i agree it's not great but it's the convention we've always used, and without it, it would make it easy for someone to add something to the signals file and not realize that it won't get imported unless you add it here.